semeio/jobs/rft/gendata_rft.py

```diff
@@ -158,18 +183,19 @@ def run(
                 well=well, time=time
             )
 
-            # Write trajectory and associated data to three  files,
+            # Write trajectory and associated data to ASCII files
             # individual pr. well and time, this is for GEN_DATA to pick up.
-            _write_gen_data_files(
-                trajectory_df,
-                os.path.join(outputdirectory, "RFT_{}_{}".format(well, report_step)),
-            )
+            _write_gen_data_files(trajectory_df, outputdirectory, well, report_step)
 
             # Aggregate dataframe for all wells and report steps.
             dframes.append(trajectory_df)
+        else:
+            logger.warning(
+                "No trajectory points for well {} at date: {} found".format(well, time)
+            )
 
-    if csvfile is not None:
+    if csvfile is not None and dframes:
         pd.concat(dframes, ignore_index=True, sort=False).to_csv(csvfile, index=None)
 
     if len(dframes) < len(well_times):
-        raise ValueError("Failed to extract requested RFT data")
+        raise ValueError("Failed to extract all requested RFT data")
```

Comment:
Should we catch these in the entry point and exit more gracefully? Still crash, but with a message instead of a traceback.

Reply:
Good idea. Changed it.